packages/compass-e2e-tests/tests/collection-bulk-delete.test.ts

```diff
@@ -172,14 +172,5 @@ filter={
 result = client['test']['numbers'].delete_many(
   filter=filter
 )`);
-
-    // Check the telemetry
-    const exportedEvent = await telemetryEntry('Delete Exported');
```

Comment:
So we're not exporting anymore? Because if the clipboard stuff?

Reply:
This event is only triggered when the copy button is clicked, there is no non-flaky way to click it in e2e after the update. I can add a more appropriate functional test for this if you feel like it's still needed

Reply:
I don't know if it is more appropriate, but if it is the only way, then I guess it will have to be a functional test.

Reply:
Pretty sure testing that you can click a React-rendered HTML button and get a callback called with e2e test is an overkill, yeah. Added a test in https://github.com/mongodb-js/compass/pull/5400/commits/8dbbabbf8ee599b98ff5ab83368c5e18acdfb571